src/main/java/com/minecolonies/client/render/TileEntityScarecrowRenderer.java

```diff
@@ -0,0 +1,104 @@
+package com.minecolonies.client.render;
+
+import com.minecolonies.client.model.ModelScarecrowBoth;
+import com.minecolonies.lib.Constants;
+import com.minecolonies.tileentities.ScarecrowTileEntity;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.renderer.tileentity.TileEntitySpecialRenderer;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+/**
+ * Class to render the scarecrow.
+ */
+@SideOnly(Side.CLIENT)
+public class TileEntityScarecrowRenderer extends TileEntitySpecialRenderer<ScarecrowTileEntity>
+{
+    /**
+     * The model of the scarecrow.
+     */
+    private final ModelScarecrowBoth model;
+
+    /**
+     * Offset to the block middle.
+     */
+    private static final double BLOCK_MIDDLE = 0.5;
+
+    /**
+     * Y-Offset in order to have the scarecrow over ground.
+     */
+    private static final double YOFFSET = 1.5;
+
+    /**
+     * Which size the scarecrow should have ingame.
+     */
+    private static final double SIZERATIO = .0625;
+
+    /**
+     * Rotate the model some degrees.
+     */
+    private static final int ROTATION = 180;
+
+    /**
+     * Rotate it on the following x offset.
+     */
+    private static final double XROTATIONOFFSET = 0.311;
+
+    /**
+     * Rotate it on the following y offset.
+     */
+    private static final double YROTATIONOFFSET = 0.0;
+
+    /**
+     * Rotate it on the following z offset.
+     */
+    private static final double ZROTATIONOFFSET = 2.845;
+
+
+    /**
+     * The public constructor for the renderer.
+     */
+    public TileEntityScarecrowRenderer() { 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) It is a good practice to call super() in a constructor [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ACallSuperInConstructor)